test/e2e/up.sh

```diff
@@ -31,7 +31,7 @@ touch ${HOME}/.kube/config
 export KUBECONFIG=${HOME}/.kube/config
 
 echo "starting Kubernetes cluster..."
-curl -Lo $DIR/dind-cluster-v1.11.sh https://raw.githubusercontent.com/kubernetes-sigs/kubeadm-dind-cluster/master/fixed/dind-cluster-v1.11.sh && \
+curl -Lo $DIR/dind-cluster-v1.11.sh https://raw.githubusercontent.com/ElvinEfendi/kubeadm-dind-cluster/fix-fixed-scripts/fixed/dind-cluster-v1.11.sh && \
```

Comment:
this is going to fix the CI, I'll revert once https://github.com/kubernetes-sigs/kubeadm-dind-cluster/pull/258 is merged